docs/apis-tools/console-api/tutorial.md

```diff
@@ -0,0 +1,263 @@
+
+## Prerequisites
+
+- If you haven't done so already, [create a cluster](/guides/assets/react-components/create-cluster.md).
+- Upon cluster creation, create your first client by navigating to **Console > Organization > Console API > Create new credentials**. Ensure you determine the scoped access for client credentials. For example, in this tutorial we will get, create, and delete a client. Ensure you check all the boxes for Zeebe client scopes.
+
+:::note
+Make sure you keep the generated client credentials in a safe place. The **Client secret** will not be shown again. For your convenience, you can also download the client information to your computer.
+:::
+
+- In this tutorial, we utilize a JavaScript-written [GitHub repository](https://github.com/camunda/camunda-api-tutorials) to write and run requests. Clone this repo before getting started.
+- Run `npm install` to ensure you have updated dependencies.
+
+## Set up authentication
+
+To get started, examine the `auth.js` file in the GitHub repository. This file contains a function named `getAccessToken` which executes an OAuth 2.0 protocol to retrieve authentication credentials based on your client id and client secret. We will call this function whenever we need an authentication token for an API request.
+
+1. To set up your credentials, create an `.env` file which will be protected by the `.gitignore` file. These keys will be consumed by the `auth.js` file to execute the OAuth protocol, and should be saved when you generate your client credentials in [prerequisites](#prerequisites).
+2. Examine the existing `.env.example` file for an example of how your `.env` file should look upon completion. You will need to add your `ADMINISTRATION_CLIENT_ID`, `ADMINISTRATION_CLIENT_SECRET`, `CLUSTER_ID`, and the `ADMINISTRATION_API_URL`, which is `https://api.cloud.camunda.io` in a Camunda 8 SaaS environment.
+
+:::note
+
+In this tutorial, we will execute arguments to view, create, and delete clients. You can examine the framework for processing these arguments in the `cli.js` file before getting started.
+
+:::
+
+## GET a list of existing clients
+
+First, let's script an API call to list our existing clients.
+
+To do this, take the following steps:
+
+1. Examine the function `async function listClients` at the top of the `administration.js` file. This is where you will script out your API call.
+2. Within the function, you must first apply an access token for this request:
+
+```
+const accessToken = await getAccessToken();
+```
+
+3. As noted in the detailed API description in [Swagger](https://console.cloud.camunda.io/customer-api/openapi/docs/#/), you must call your Administration API URL and cluster ID. Ensure these credentials are added to your `.env` file, and script the references to the values. For example:
+
+```
+const administrationApiUrl = process.env.ADMINISTRATION_API_URL;
+  const clusterId = process.env.CLUSTER_ID;
```

Comment:
There are some indentation discrepancies here and in other code blocks -- I think a result of the fact that in the code repo, they're indented another level, but here we're left-aligning them either intentionally or because of prettier. Which do you think we should align to -- zero indentation, or what is in the code repo?

Reply:
Let's go with the code repo 👍 I'll adjust.